site/content/extend/plugins/community_process.md

```diff
@@ -58,6 +58,7 @@ QA pass involves getting a memeber of our QA team to take a look and verify the
 
 Steps involved:
 - Ensure all setup documentation needed is clear and can be successfully followed 
+- Dedicated instance for a third-party service is deployed, if applicable
```

Comment:
I'm not sure I understand....what's an example?  Like a test account for Service X?

Reply:
An example is an Azure or Microsoft account with a running on-prem Skype4Business server instance to review and test Skype4Business plugin. I tested the cloud version with a one-off set up for the bounty, but if we want to support it officially, it'll be more efficient to set up an instance that we can continuously test on.

Reply:
```suggestion
- Dedicated instance or test account to access and test the third-party service, if applicable
```